packit_service/worker/handlers/copr.py

```diff
@@ -587,6 +591,16 @@ def handle_scan(self):
                 ),
             )
 
+    @staticmethod
+    def parse_dict_from_output(output: str):
+        json_pattern = r"\{.*?\}"
+        matches = re.findall(json_pattern, output, re.DOTALL)
+
+        if matches:
+            json_str = matches[-1]
+
+        return json.loads(json_str)
```

Comment:
`json_str` is only defined if there the regular expression matches. This line would throw an exception if `json_str` is not defined.